lib/ArtistClass.ts

```diff
@@ -1,17 +1,29 @@
+import { nanoid } from "nanoid";
+
 function stringify(string: string): string {
   const noBlanks = string.replaceAll(" ", "");
   return noBlanks.toLowerCase();
 }
 
```

Comment:
`stringify` sounds strange if the input is already a string... 
Maybe you could call it `removeWhitespace` of `normalize` or sth. similar?
You could also make it a oneliner: `return string.replaceAll(" ", "").toLowerCase()`

Reply:
Good catch, I will change that!